### PR TITLE
docs(website): roadmap update

### PR DESCRIPTION
**Pull request by aayushpi**
- Remove Component Lifecycle
- Simplify Roadmap Structure
- Update Content

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [X] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by SiTaggart** [comment]
Given there is only one 2 week sprint left I don’t think it’s realistic to have 

- Select
- Checkbox and checkbox group
- Radio and radio group
- modal 

Done in that time. Checkbox design hasn’t even started yet, and I don’t think Modal is finished. Hopefully it will be.

It’s probably fine for sketch deliverables, but not React.

File: packages/paste-website/src/pages/roadmap/index.mdx [hunk 1]
```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
...
-Release date: February 25, 2020
+  ## Upcoming Release
 
-**Design releases**
+  Paste's upcoming release will be on **April 7, 2020**. This is what we've planned:
 
-- Design more form field components (Select, Checkbox and Checkbox group, and Radio and Radio Group). 📝
-- Design a Form composition component. 📝
-- Design a Toast component. 🍞
+  - Form components are the bread and butter for most Twilio experiences. With this release we'll ship seven components (Text Input, Textarea, Select, Checkbox, Checkbox Group, Radio, and Radio Group) for Sketch and React. 📝
```
Comment (serifluous) [inline packages/paste-website/src/pages/roadmap/index.mdx]: "Form components are the bread and butter for most Twilio experiences. With this release we'll ship three form components (Text Input, Textarea, Select) for Sketch and React."

File: packages/paste-website/src/pages/roadmap/index.mdx [hunk 2]
```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
...
-- Design a Form composition component. 📝
-- Design a Toast component. 🍞
+  - Form components are the bread and butter for most Twilio experiences. With this release we'll ship seven components (Text Input, Textarea, Select, Checkbox, Checkbox Group, Radio, and Radio Group) for Sketch and React. 📝 
 
-**Engineering releases**
+  - We'll ship a Content guide for Twilio UX, which should help you to feel more confident in writing concise, clear, and consistent product content. 🖊 
 
-- Publish the Modal Primitive component
-- Publish a Grid layout component
+  - Modals, an important container element to focus user attention, will be shipped for Sketch and React. 🔳
```
Comment (serifluous) [inline packages/paste-website/src/pages/roadmap/index.mdx]: Maybe? Are building both Select and Modal viable with eng capacity?

File: packages/paste-website/src/pages/roadmap/index.mdx [hunk 3]
```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
...
+  ---
 
-## Release 2020.04.07
+  ## Future Releases
 
-Release date: April 7, 2020
+  Future releases of Paste are scheduled for every six weeks, with the next release on **May 19, 2020**. Our guidance on what's planned will update as we get closer to the date. For now, here's what we're hoping to do:
 
-**Design releases**
+  - Ship a Form composition component. 📝
```
Comment (serifluous) [inline packages/paste-website/src/pages/roadmap/index.mdx]: Ship three more form components (Checkbox, Checkbox Group, and Radio Group) for Sketch and React." 📝

File: packages/paste-website/src/pages/roadmap/index.mdx [hunk 4]
```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
...
+  - Ship a Toast component. 🍞
+  - Add an icon list to the [icon system guidelines](/icon-system) with more design guidelines. 👩🏽‍🎤
```
Comment (serifluous) [inline packages/paste-website/src/pages/roadmap/index.mdx]: Potentially add Combo-box primitive, Stack utility.

**Pull request merged**